Source/Renderer/UniformState.js

```diff
@@ -148,6 +148,10 @@ define([
         this._resolutionScale = 1.0;
 
         this._fogDensity = undefined;
+
+        // TODO : expose these differently
```

Comment:
I think it would be fine to just have a `ShadowMap` reference here, and then assign to it each frame when calling `UniformState.update` (pass it as a separate parameter to start, I guess).